bentoml/_internal/frameworks/transformers.py

```diff
@@ -18,7 +18,8 @@
 from ..utils.pkg import get_pkg_version
 from ..configuration.containers import BentoMLContainer
 
-logger = logging.getLogger(__name__)
+logging.captureWarnings(True)
```

Comment:
what is this for?

Reply:
redirect warnings capture from `jax` to logger